examples/miscellaneous/opacitywindow.py

```diff
@@ -0,0 +1,28 @@
+from kivy.app import App
+from kivy.lang import Builder
+
+kv = '''
+#:import window kivy.core.window.Window
+
+BoxLayout:
+    orientation: 'vertical'
+    Label:
+        text: f'Window opacity: {window.opacity}'
+        font_size: 25
```

Comment:
Add `sp` units to font size. Change this line to: `font_size: '25sp'`.

Reply:
Done.